vagrant/Makefile

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/make -f
+
+cluster = microk8s
+endpoint = http://receiver-mock.receiver-mock:3000/terraform/api/
+access_id = dummy
+access_key = dummy
+name = collection
+mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
```

Comment:
Out of curiosity why do we need this construct?

Reply:
I wanted this Makefile to be executable, so need to get absolute path of the file and later abs path of the repository. I will add comments to clarify that